pkg/validate/security_context_linux.go

```diff
@@ -212,6 +212,7 @@ var _ = framework.KubeDescribe("Security Context", func() {
 
 			By("get nginx container pid")
 			command := []string{"cat", "/proc/1/cmdline"}
+			time.Sleep(time.Second) // waits for nginx to be up-and-running
```

Comment:
I'm curious why this test case fixed with the delay when the next occurrence is  fixed with the re-tryable command?

Reply:
I think he wants this one to fail :-)

Reply:
Yes, this test expects `ToNot(ContainSubstring("master process"))` so I thought retrying this until timeout will result in the very slow test.